package/restream/package

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(restream)
+pkgdesc="A binary framebuffer capture tool for the reStream script"
+url=https://github.com/rien/reStream
+pkgver=0.0.0-c41b877
+timestamp=2021-01-01T14:55:28+01:00
```

Comment:
This should be set to the date of https://github.com/rien/reStream/commit/c41b87778953513cd52d9c58b3cc5ce52825700e with no timezone modifier.

Reply:
That is the commit date according to `git log --date=iso-strict`. I'll fix the TZ, but I avoided changing it because I know I'll get that math exactly wrong :)

Reply:
Setting the timestamp is the one thing I really dislike doing in the packages myself due to this :P

Reply:
Done, used [this](https://dencode.com/en/date/iso8601).